buzzmobile/sense/inputer/inputer.lisp

```diff
@@ -0,0 +1,7 @@
+(in-package :buzzmobile)
+
+(defun inputer ()
```

Comment:
Actually though, the original node was also named 

> inputer

So if it's a bad name we should defs change it